plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -1,6 +1,18 @@
 <script type="text/javascript">
     $(document).ready(function(){
-        $('.tabs').tabs();
+        $('.tabs').tabs({
+            'onShow': function (element) {
+                var id = element.getAttribute('id');
+                if (!id) {
+                    return;
+                }
+                if(history.pushState) {
+                    history.pushState(null, null, '#' + id);
+                } else {
+                    window.location.hash = id;
+                }
+            }
+        });
```

Comment:
That actually only works half the way. If there is already a hash in the URL it will be simply thrown away and replaced with the tabs id. And if the page is loaded with a certain hash, the tab won't be automatically preselected.
I've tried locally what would work best, and I think this approach would be what I would go with:
```suggestion
        $('.tabs').tabs({
            'onShow': function (element) {
                var id = element.getAttribute('id');
                if (!id) {
                    return;
                }
                window.CoreHome.MatomoUrl.updateHash({...window.CoreHome.MatomoUrl.hashParsed.value, tab: id})
            }
        });
        var selectedTab = window.CoreHome.MatomoUrl.hashParsed.value.tab;
        if (selectedTab) {
            M.Tabs.getInstance($('.tabs')).select(selectedTab);
        }
```

Reply:
@sgiehl Thanks, I have applied the suggestion :+1: